main.go

```diff
@@ -19,11 +19,13 @@ import (
 )
 
 var (
-	allowedHosts argumentList
-	imaginaryURL string
-	listenPort   int64
-	bucketRate   float64
-	bucketSize   int64
+	allowedHosts            argumentList
+	allowedImaginaryParams  string
+	allowedImaginaryActions string
```

Comment:
I was a bit in doubt about the `argumentList` thing. Repeated arguments made sense for domains, but I don't like it for parameters or actions. But I like a different type that generalises comma-separated values. :+1: